content/vac/dst/deployment-and-analysis/waku/midscale.md

```diff
@@ -40,13 +40,23 @@ Testing is to be done in this order of priority:
 4. Test Store protocol at scale
 5. Test Waku relay+store reliability with nodes going offline/online
   - If nodes go online/offline, we should be able to retrieve missing messages from the store. This will also test Waku message relaying in a different way.
-6. Filter and lightpush tests
-7. Measure (1) and (3) in heterogenous clusters involving different node implementations such as nwaku and go-waku
-8. Measure (3) with Waku peer exchange protocol used for discovery by a subset of nodes.
-9. Measure (1) with a mix of nodes using Resource-restricted device reliability protocol and peer exchange, meaning a small number of nwaku nodes serve store, light push and filter protocols and a high number of clients consume them. For example, 6-10 service nodes, 200 relay nodes and 1000 light nodes.
+6. Measure (1) and (3) in heterogenous clusters involving different node implementations such as nwaku and go-waku
+7. Test waku shard behaviour and stability with various of numbers of shards
+8. Filter and lightpush tests
+9. Measure (3) with Waku peer exchange protocol used for discovery by a subset of nodes.
+10. Measure (1) with a mix of nodes using Resource-restricted device reliability protocol and peer exchange, meaning a small number of nwaku nodes serve store, light push and filter protocols and a high number of clients consume them. For example, 6-10 service nodes, 200 relay nodes and 1000 light nodes.
 This should include connection and node churn impact on reliability for both relay and light clients.
 
+Additionally, perform monthly regression tests against a chosen version of Waku to ensure that no new bugs have been introduced. Produce a report on the results of the tests and any interesting findings.
+
 ### Justification
+Provide a greater understanding of Waku's performance and reliability in midsized networks.
 
 ### Deliverables
-Similar deliverables to 10k sim, but with focuses on smaller scale and more frequent deployments.
\ No newline at end of file
+Frequent deployments of 1000-5000 node networks.
```

Comment:
If we want to add 4000 and 5000 nodes results, this will substantially increase the simulation times and analysis.
We can do 1K-3K-5K. But for 1msg/s we will probably need some tweaking. 
I am not sure if that scenario right now can be handled.

Reply:
Yeah, this wasn't meant to imply that we'd do those every time - more just a general statement to cover the milestone (since 1000-5000 = midscale)